.github/workflows/build.yml

```diff
@@ -101,9 +101,9 @@ jobs:
           fi
         done
 
-  build:
+  release:
     runs-on: ubuntu-latest
-    needs: [test, lint, examples]
+    needs: [test, lint, examples, license-check]
```

Comment:
We also need to restore the `if: startsWith(github.ref, 'refs/tags/')` to ensure that the step is only triggered on a push of a `semver` tag.

Reply:
added tag condition to release job